app/components/molecules/CustomMolecule/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { View, Image, Text } from 'react-native';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import { fonts, images } from '@themes';
```

Comment:
awesome, 
we try to import stuffs the same way as in this file, 
```
// 3rd part
// alias imports
// relative imports

```
just one nit:-
lets have a space between the imports, i personally prefer this way, makes it look easier to understand

```

import 3rd party
<blank space>
alias imports
<blank space>
relative imports
<blank space>